packages/module/patternfly-docs/content/extensions/data-view/examples/Toolbar/Toolbar.md

```diff
@@ -43,7 +68,7 @@ The `useDataViewPagination` hook manages the pagination state of the data view.
 - optional `searchParams` object
 - optional `setSearchParams` function
 
-While the hook works seamlessly with React Router library, you do not need to use it to take advantage of URL persistence. The `searchParams` and `setSearchParams` props can be managed using native browser APIs (`URLSearchParams` and `window.history.pushState`) or any other routing library of your choice. If you don't pass these two props, the pagination state will be stored internally without the URL usage.
+While the hook works seamlessly with the React Router library, you do not need to use it to take advantage of URL persistence. The `searchParams` and `setSearchParams` props can be managed using native browser APIs (`URLSearchParams` and `window.history.pushState`) or any other routing library of your choice. If you don't pass these two props, the pagination state will be stored internally without the URL usage.
 
 You can also pass custom `pageParam` or `perPageParam` names, renaming the pagination parameters in the URL.
 
```

Comment:
I'm unable to make suggestions on the following lines, but here's what I would do: 

The retrieved values are named to match the [PatternFly pagination](/components/pagination) component props. 

**Return values:**
- Current `page` number
- `onSetPage` to modify current page
- Items `perPage` value
- `onPerPageSelect` to modify per page value

### Pagination example
This example uses the URL to persist the pagination state.